src/test/miner_tests.cpp

```diff
@@ -378,7 +380,7 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     tx2.vin[0].scriptSig = CScript() << OP_1;
     tx2.vin[0].nSequence = 0;
     tx2.vout.resize(1);
-    tx2.vout[0].nValue = 3900000000LL;
+    tx2.vout[0].nValue = 790000LL;
     tx2.vout[0].scriptPubKey = CScript() << OP_1;
```

Comment:
This change and the one immediately above (on added line 369) fix [this failing test](http://ci.leastauthority.com:8010/builders/zcash/builds/144/steps/full-test-suite.sh/logs/stdio). But I had forgotten to make all the equivalent changes further up, and the test still passed. I am therefore unsure if the test is invalid because it is not testing the right behaviour, or if the test is fine but unclear because it is testing behaviour that does not rely on having valid transactions.